services/web/server/tests/integration/test_exporter.py

```diff
@@ -0,0 +1,346 @@
+    yield
+
+
+@pytest.fixture(autouse=True)
+async def __delete_all_redis_keys__(redis_service: RedisConfig):
+    client = await aioredis.create_redis_pool(redis_service.dsn, encoding="utf-8")
+    await client.flushall()
+    client.close()
+    await client.wait_closed()
+
+    yield
+    # do nothing on teadown
+
+
+@pytest.fixture
+async def monkey_patch_aiohttp_request_url() -> None:
+    old_request = aiohttp.ClientSession._request
+
+    async def new_request(*args, **kwargs):
+        assert len(args) == 3
+
+        url = args[2]
+        if isinstance(url, str):
+            url = URL(url)
+
+        if url.host == "director-v2":
+            from pytest_simcore.helpers.utils_docker import get_service_published_port
+
+            log.debug("MOCKING _request [before] url=%s", url)
+            new_port = int(get_service_published_port("director-v2", 8000))
+            url = url.with_host("172.17.0.1").with_port(new_port)
+            log.debug("MOCKING _request [after] url=%s kwargs=%s", url, str(kwargs))
+
+            args = args[0], args[1], url
+
+        return await old_request(*args, **kwargs)
+
+    aiohttp.ClientSession._request = new_request
+
+    yield
+
+    aiohttp.ClientSession._request = old_request
+
+
+@pytest.fixture
+def client(
+    loop, aiohttp_client, app_config, postgres_with_template_db, mock_orphaned_services
+):
+    cfg = deepcopy(app_config)
+
+    assert cfg["rest"]["version"] == API_VERSION
+    assert cfg["rest"]["enabled"]
+    cfg["projects"]["enabled"] = True
+    cfg["director"]["enabled"] = True
+
+    # fake config
+    app = create_safe_application(cfg)
+
+    # activates only security+restAPI sub-modules
+    setup_db(app)
+    setup_session(app)
+    setup_security(app)
+    setup_rest(app)
+    setup_login(app)
+    setup_users(app)
+    setup_socketio(app)
+    setup_projects(app)
+    setup_director(app)
+    setup_director_v2(app)
+    assert setup_resource_manager(app)
+
+    yield loop.run_until_complete(
+        aiohttp_client(
+            app,
+            server_kwargs={"port": cfg["main"]["port"], "host": cfg["main"]["host"]},
+        )
+    )
+
+
+################ utils
+
+
+async def login_user(client):
+    """returns a logged in regular user"""
+    return await log_client_in(client=client, user_data={"role": UserRole.USER.name})
+
+
+def get_exported_projects() -> List[Path]:
+    exporter_dir = CURRENT_DIR / ".." / "data" / "exporter"
+    return [x for x in exporter_dir.glob("*.osparc")]
+
+
+@pytest.fixture
+async def mock_asyncio_subporcess(mocker):
+    # TODO: The below bug is not allowing me to fully test,
+    # mocking and waiting for an update
+    # https://bugs.python.org/issue35621
+    # this issue was patched in 3.8, no need
+    if sys.version_info.major == 3 and sys.version_info.minor >= 8:
+        raise RuntimeError(
+            "Issue no longer present in this version of python, "
+            "please remote this mock on python >= 3.8"
```

Comment:
3.8 is coming...

Reply:
I am aware, waiting for the PR to merge it